README.md

```diff
@@ -203,7 +203,7 @@ Note in most cases, the err.Details() will be used to generate replacement strin
 ```
 
 ## Formats
-JSON Schema allows for optional "format" property to validate strings against well-known formats. gojsonschema ships with all of the formats defined in the spec that you can use like this:
+JSON Schema allows for optional "format" property to validate strings and numbers against well-known formats. gojsonschema ships with all of the formats defined in the spec that you can use like this:
```

Comment:
maybe change "strings and numbers" to "instances" (to be consistent w/ terminology spec uses)

Reply:
I agree that "instances" is preferable, although it should match the actual support.  if the next step is just to get numbers as well as strings, that would be good (there are definitely numeric format suggestions, but while I can make up some formats for other types none are actually motivated by use cases :-)